doc/source/whatsnew/v0.18.2.txt

```diff
@@ -318,3 +319,5 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+- Bug in ``DataFrame.describe()`` where percentile identifiers in index were always rounded to at most 1 decimal place (:issue:`13104`)
```

Comment:
can you move all of theses describe issues to a separate sub-section (``.describe() changes``), where you show an example of the previous behavior (e.g. give the example from the original issue), and then show the new way of displaying it.

Even though these are 'bug' fixes, a user would want to know that this is happening.